src/fi_tostr.c

```diff
@@ -44,6 +44,7 @@
 #include <arpa/inet.h>
 
 #include "ofi.h"
+#include "shared/ofi_str.h"
```

Comment:
This isn't a result of your changes and a rhetorical question, but why is this header under a shared subdirectory?

Reply:
You pointed out before that this shared/xxx is shared between libfabric and fi_info, fi_pingpong.  Is this still a valid case?  Otherwise, I can remove the subdirectory of shared.

Reply:
I don't think we need include/shared if you want to remove it.  AFAICT, only libfabric uses that file.  It's also used by providers, but the providers also just make use of files directly under include.  So, there's no benefit there.

The src/shared file used to be valid, as that was shared between libfabric and providers (built as DL).  But that's also no longer the case, as the providers now build with files directly under src.

If you can remove the include/shared and src/shared subdirectories, I think that will simplify things for now.  Maybe we can do a more in depth source restructuring later of files under src and prov/util.